### ISSUE TITLE
For October 2017

### ISSUE THREAD

**Issue by cbeams** [bisq-network/compensation#1]
## Basic information

> _Specify the amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ amount requested: ~~9000~~ **14000** (updated per comment thread below)
 - BSQ address: **B1HGxmhfWJwwvvDgTRrWpZ1QsEZn7rMEaVE**

## Details

> _Provide links to the work you are requesting compensation for, along with any comments or explanations that will help stakeholders understand its value. **Please include only completed work!**_

### General notes
For me, this month was all about preparing the _Phase Zero_ plan for bootstrapping the Bisq DAO. This work started in September, and culminated with the delivery of a talk at Hackers Congress 2017 in Prague, followed two weeks later by the publication of the Phase Zero paper. The paper got some attention via Twitter, and video of the conference talk will be published soon. I think the real value of the paper will be in the months to come, as we put the plan into practice, refine it, and use it to attract and onboard new contributors.

Regarding the amount of BSQ I'm requesting here (9,000), this is a tough call, especially since it's the first time we're doing this. Here are a few of the things that have made this process challenging for me:

 1. We have not yet finished the exercise of establishing an initial fair market value for BSQ, so we are simply going with 1 BSQ == 1 USD this month. This is not just an arbitrary number; it is what Manfred has had in mind for many months now. It's beyond the scope of this note to go into much further detail, but just keep in mind that this value *will* change in the future as we refine the fair market value model. Hopefully that value won't be too far from the 1 USD mark, but we'll just have to see how it goes.

 2. I (like many others) am used to getting paid a monthly salary, basically for showing up and "doing my job". Within the Bisq DAO, though, I think we need to take a different approach, wherein we get paid not simply for the work we do or effort we put in, but rather _for the results we produce_. Working on a feature doesn't become valuable to the network until that feature has been shipped (or at least until it has been merged into the mainline and is _ready_ to ship). Working on a paper doesn't become valuable until the paper has been published, and so on. We have to think like future BSQ stakeholders who are voting on compensation requests and asking themselves whether what they see is worth inflating the BSQ supply for. If we try to stick to requesting compensation only for results, i.e. completed work, I think it'll make the voting process that much easier. In any case, I'm sure there will be more discussion on this front, but I wanted to get the ball rolling here.

 3. As a co-founder of the project, it's hard to determine whether I should ask for my "market rate", i.e. what I believe I would have been paid by other organizations for this work, or whether I should ask for more modest compensation, as founders often do in a startup environment. My market rate would have been somewhere around 14,000 USD (and therefore 14,000 BSQ). This is based on the amount of responsibility I have here in the Bisq DAO and how much I've been paid in previous positions where I had similar levels of responsibility. In the end, I've taken what I think is a more conservative approach here and requested 9,000 BSQ because it feels to me like a fair valuation of the results I've produced this month, while also taking into account that a key goal of Phase Zero is to redistribute stake away from founders to other contributors. Again, I'm sure we'll have more conversation on this front over the months to come, and I reserve the right to change what I ask for in future compensations based on those conversations. It'll be a process to find the right compensation levels; it'll be an important part of the culture that we build.

So much for my general notes; now on to the specifics...

### HCPP17 conference talk
Video of the talk has not yet been published, but will be soon at https://www.youtube.com/channel/UCfHJ5Y3akQ7LA0PQmSYlYmQ. In the meantime, here's a screenshot of the abstract of my talk from https://liberate.hcpp.cz/schedule:

![image](https://user-images.githubusercontent.com/301810/32142451-abdd2fd2-bc97-11e7-863f-c7f8ea284a43.png)

And here are the slides I used for the talk. They should be quite useful for future talks as well: https://docs.google.com/presentation/d/1G5_6Kju2OoItZ5lD0jnskqLdOV2dAJpdB6WOjmq9hRk/edit#

The talk was well attended at the conference, and I got considerable positive feedback about it from attendees.

Travel costs for the trip were minimal, basically just a round trip train ticket and a handful of Uber rides; the BSQ request amount above accounts for this.

### WCN interview
While at HCPP17, I gave the following 15-minute presentation with World Crypto Network, covering Bisq, the Bisq DAO, and the Phase Zero plan: https://www.youtube.com/watch?v=AvXWToT-dcQ. The video has had about 3K views so far.

### Phase zero paper
The Phase Zero paper itself is available at: https://github.com/bisq-network/docs/blob/master/dao/phase-zero.adoc. Many thanks to those who helped review it.

Here's a tweetstorm announcing the paper: https://twitter.com/bisq_network/status/921551113463222274

And the forum announcement: https://forum.bisq.io/t/phase-zero-a-plan-for-bootstrapping-the-bisq-dao/3200

And some great feedback on Twitter: https://twitter.com/cburniske/status/922549242245328898

Note that there will be additional announcements via the Bisq newsletter soon. We've basically only announced via Twitter and the Forum at this point.

### Orchestrating the first round of compensation requests and voting
Including:
 - setting up this repository and designing compensation request issues like this one
 - the spreadsheet we're using to vote on them
 - and the communications to let everyone know about it

### Twitter handle operations

Per bisq-network/roles#21, I continued operating the @bisq_network Twitter handle this month. In the future, we'll probably start doing reporting for these roles on their role issues, complete with analytics updates, etc. This sort of thing becomes much more feasible when people are only responsible for one or two roles; it's too much work for me and Manfred to do this sort of thing right now, while we're still responsible for most roles.

**Comment by ManfredKarrer** [comment]
I have a few comments.

Regarding point 1 in your General notes:
I used 1 EUR as orientation price for 1 BSQ in the past. After discussing we agreed to use USD as that is more standard in project valuations. 

Regarding point 2 in your General notes:
I agree that the results and deliveries should be taken as basis, but it might be difficult for those who work on many different smaller things and not all is documented, or if it is, it will be still hard to estimate or extract a total value out of it (we don't want overhead with documenting every tiny task). As a simple way to reference to all the small items the Github activity of the requester can be used if Github was used mainly as the organizational tool. 
Most contributors will have anyway a more clearly defined work scope (issues, PR,...) so for those this problem will be less relevant. At the moment it is mainly myself and Chris who are doing lots of stuff cross boundaries and I think the request should cover also that work which is not clearly expressed in "deliveries".

Regarding point 3 in your General notes:

I think we should not introduce "discounts" - however they are motivated (founders, redistribute stake, open source project,...). If someone start adding such it will create also pressure for others to do that as well. I think it is easier and more clear to use the estimated market value of the work only. In your case I think the 14 000 is more appropriate and covers much work not see so clearly as the phase zero doc.

I also think it is not expose past salaries. Thought that can be used as personal orientation point to check the market value of one's work in general.

**Comment by cbeams** [comment]
> Regarding point 1 in your General notes:
> I used 1 EUR as orientation price for 1 BSQ in the past. After discussing we agreed to use USD as that is more standard in project valuations.

Right, we'll denominate the value of BSQ in USD from here out. Thanks for mentioning.

> Regarding point 2 in your General notes:
> I agree that the results and deliveries should be taken as basis, but it might be difficult for those who work on many different smaller things and not all is documented, or if it is, it will be still hard to estimate or extract a total value out of it (we don't want overhead with documenting every tiny task). As a simple way to reference to all the small items the Github activity of the requester can be used if Github was used mainly as the organizational tool.
> Most contributors will have anyway a more clearly defined work scope (issues, PR,...) so for those this problem will be less relevant. At the moment it is mainly myself and Chris who are doing lots of stuff cross boundaries and I think the request should cover also that work which is not clearly expressed in "deliveries".

Yeah, I'd like to underscore this point, too, that we do not want to create a bureaucracy where every little task has to be documented in detail. I think your compensation request (#2) strikes the right balance. Just provide a few links to stuff you got done over the month, and add any comments you think might be necessary. People can always ask follow-up questions in the comment thread if more detail is necessary. We're still a small group of contributors, so it should be pretty easy for folks to catch up on what others have been doing.

> Regarding point 3 in your General notes:
> 
> I think we should not introduce "discounts" - however they are motivated (founders, redistribute stake, open source project,...). If someone start adding such it will create also pressure for others to do that as well. I think it is easier and more clear to use the estimated market value of the work only. In your case I think the 14 000 is more appropriate and covers much work not see so clearly as the phase zero doc.

Yeah, thanks. I really struggled with that, and wasn't very happy with it in the end. I've updated the requested BSQ amount from 9000 to 14000. This is actually a good example of how the compensation request and voting process should go in the future. We have to be unafraid to say "no" not only to requests that are too high, but also to requests that are too low.
 
> I also think it is not expose past salaries. Thought that can be used as personal orientation point to check the market value of one's work in general.

I agree, and I was just trying to put my thought process on loudspeaker for whatever value it might add to other contributors in this early phase. To be super clear: people do _not_ need to justify how much they are asking for based on previous salaries. You can keep that information private, even though I chose not to!

**Comment by cbeams** [comment]
I've entered this compensation request into the [stake and vote tracking spreadsheet](https://docs.google.com/spreadsheets/d/1xlXDswj3251BPCOcII-UyWlX7o7jMkfYBE-IZ5te5Ck/edit#gid=0).

**Comment by sqrrm** [comment]
The data entered in the sheet doesn't look right for issue2, inverted requester voter rows.

**Comment by cbeams** [comment]
> The data entered in the sheet doesn't look right for issue2, inverted requester voter rows.

Good catch, thank you. That was my mistake.

@ManfredKarrer, I noticed that you voted on my and your requests already. I removed those votes, as it's probably best to wait until the voting period actually begins (which is November 6th).

Perhaps the better thing, though, is just for me to wait to enter things into the sheet at all until the day the voting period begins. That way there is nothing for people to vote early on.
